File: app/src/main/java/com/example/pearvideoclient/utils/ScreenUtils.java
package com.example.pearvideoclient.utils;

import android.content.Context;

/**
 * @Description: java类作用描述
 * @Author: jiazhu
 * @Date: 2019-08-23 21:39
 * @ClassName: ScreenUtils
 */
public class ScreenUtils {
    private ScreenUtils() {
        // do nothing
    }

    public static int dip2px(Context context, float dipValue) {
        float scale = context.getResources().getDisplayMetrics().density;
        return (int) (dipValue * scale + 0.5F);
    }
}

File: app/src/main/java/com/example/pearvideoclient/utils/ScreenUtils.kt
package com.example.pearvideoclient.utils

import android.content.Context

/**
 * @Description: java类作用描述
 * @Author: jiazhu
 * @Date: 2019-08-23 21:39
 * @ClassName: ScreenUtils
 */
object ScreenUtils {
    @JvmStatic
    fun dip2px(context: Context, dipValue: Float): Int {
        val scale = context.resources.displayMetrics.density
        return (dipValue * scale + 0.5f).toInt()
    }
}
